src/app.js

```diff
@@ -0,0 +1,62 @@
+import isYes from '../src/is-yes.js';
+
+const quizButton = document.getElementById('quiz-button');
+const result = document.getElementById('result');
+
+quizButton.onclick = function() {
+    //console.log('working??');
```

Comment:
In the future, you'll want to delete out any commented out code before you submit. Starting next week, we'll start taking points off for any commented code left in your submission. If you're worried about losing comments, just commit before you delete them so that you can refer back to the comments on that commit. 

Other than that, your code looks really clean! I didn't notice any unused code, and everything looks really good.